objcgen/embedder.cs

```diff
@@ -290,7 +290,7 @@ public int Compile ()
 					break;
 				case Platform.iOS:
 					build_infos = new BuildInfo[] {
-					new BuildInfo { Sdk = "iPhoneOS", Architectures = new string [] { "armv7", "armv7s", "arm64" }, SdkName = "iphoneos", MinVersion = "8.0", XamariniOSSDK = "MonoTouch.iphoneos.sdk" },
+					new BuildInfo { Sdk = "iPhoneOS", Architectures = new string [] { "armv7", "armv7s", "arm64" }, SdkName = "iphoneos", MinVersion = "8.0", XamariniOSSDK = "MonoTouch.iphoneos.sdk", CompilerFlags = "-fembed-bitcode", LinkerFlags = "-fembed-bitcode" },
```

Comment:
I wonder if we should make the inclusion of `CompilerFlags = "-fembed-bitcode", LinkerFlags = "-fembed-bitcode"` conditional to be enabled by a flag or by the presence of `Extension`, not everyone would like to have bitcode since it bloats the libraries specially for AdHoc/Enterprise deployments that are off limits of the AppStore.